Make send mail command

crowity opened the pull request:
Make send mail command ( !sendmail [uid] [text] [sender] / optional [itemid] [amount] etc.

4Benj committed:
Log files

memetrollsXD committed:
Merge branch 'stable' into development

fumbling644o committed:
Explicitly load and save UTF-8 for the handbook

This fixes potential output issues for non-Latin scripts when exporting the handbook by changing the filename in the code.

Commit:
Merge pull request #107 from memetrollsXD/development

Keep development branch up to date with stable branch

Commit:
Merge pull request #104 from 4Benj/development

Log files

4Benj commented:
Why tf is this a PR and why is it merging dev into stable?

memetrollsXD reviewed (changes requested):
This PR is so scuffed.. My pull request merging dev to main was already accepted, so this will just cause duplicate commits and trouble. This also doesn’t add anything of a “mail” command and is just merging branches

alt3ri committed:
Added /talent command

Set current character talent level by using /talent command.
> /talent getid - Gets current character talent ID
> /talent set <talent ID> <level> - Sets current character's talent level

alt3ri committed:
Deleted 2 lines

Commit:
Merge pull request #108 from fumbling644o/development

GM handbook: Explicitly load and save UTF-8

Commit:
Merge pull request #114 from alt3ri/development

Added /talent command

Melledy commented:
This should be made as an issue, not a PR.
